SwiftyContextMenu/ContextMenuActionTableViewCell.swift

```diff
@@ -68,14 +68,17 @@ class ContextMenuActionTableViewCell: UITableViewCell {
     private func addSeparatorView() {
         let separatorView = ContextMenuSeparatorView(frame: .zero, style: self.style)
         self.separatorView = separatorView
-        addSubview(separatorView)
         separatorView.translatesAutoresizingMaskIntoConstraints = false
-        NSLayoutConstraint.activate([
-            separatorView.heightAnchor.constraint(equalToConstant: 0.33),
-            separatorView.topAnchor.constraint(equalTo: topAnchor),
-            separatorView.leadingAnchor.constraint(equalTo: leadingAnchor),
-            separatorView.trailingAnchor.constraint(equalTo: trailingAnchor)
-        ])
+        contentView.addSubview(separatorView)
+        DispatchQueue.main.async {
```

Comment:
Sync with the next frame otherwise it's too soon to set constraints for the cell and this warning's pop in logs.

```
[Warning] Warning once only: Detected a case where constraints ambiguously suggest a height of zero for a table view cell's content view. We're considering the collapse unintentional and using standard height instead. Cell: <SwiftyContextMenu.ContextMenuActionTableViewCell: 0x7fd698c1e7b0; baseClass = UITableViewCell; frame = (0 0; 250 44); text = 'Looooooooooooong title'; autoresize = W; layer = <CALayer: 0x600002ca11e0>>
```
As other solutions we could also: 
- Move the `NSLayoutConstraint.activate` in `layoutSubviews()` (but I think it will be less efficient).
- Abandon auto-layout for frame-layout
- Other solution that you may think of ?

Reply:
I like this solution 👍